### PR TITLE
Package Updates

### PR DESCRIPTION
**Pull request by ravish1729**
Adding package to lighthouse organization
New user bug fix
balance-stable coins added

### CODE REVIEW

File: Commands/balance.js [hunk 1]
```diff
@@ -47,6 +50,76 @@ module.exports = {
...
+            chalk.yellow("USDC Balance: ") +
+              Array(2).fill("\xa0").join("") +
+              ethers.utils.formatUnits(
+                await contractUSDC.balanceOf(
+                  config.get("LIGHTHOUSE_GLOBAL_PUBLICKEY")
+                ),
+                lighthouseConfig[network]["usd_contract_decimal"]
+              )
+          );
+
```
Comment (nandit123) [inline Commands/balance.js:109]: cant see the code for DAI, we should support that too

**Review by nandit123 — CHANGES_REQUESTED** [review]
cant find the code for DAI

File: Commands/balance.js (hunk 1 above)
Comment (ravish1729) [reply Commands/balance.js:109]: Added

**Review by nandit123 — COMMENTED** [review]
lgtm

**Pull request merged**